iothub/service/samples/how to guides/CleanupDevicesSample/ImportError.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Samples
+{
+    internal class ImportError
```

Comment:
I'm thinking about adding this to the SDK for v2, like we have for ImportExportDevice. Thoughts?

Reply:
I'm surprised to hear we don't already have this feature. I'm in favor of adding it, sure.